src/modules/graphics/Font.cpp

```diff
@@ -896,14 +896,11 @@ void Font::getWrap(const ColoredCodepoints &codepoints, float wraplimit, std::ve
 	}
 
 	// Push the last line.
-	if (!wline.cps.empty())
```

Comment:
We'll need to see if anything special needs to be done for generated newlines (as opposed to explicitly added newlines) there, I suppose. I suspect that was the reason for that conditional in the first place, but I'm not sure if it's actually necessary or not.